pkg/ckecli/cmd/vault_init.go

```diff
@@ -59,6 +60,30 @@ path "cke/*"
 }`
 )
 
+func readPasswordFromStdTerminal(prompt string) (string, error) {
+	if !term.IsTerminal(int(os.Stdin.Fd())) || !term.IsTerminal(int(os.Stdout.Fd())) {
+		return "", fmt.Errorf("stdin and stdout are not terminals")
+	}
+
+	screen := struct {
+		io.Reader
+		io.Writer
+	}{os.Stdin, os.Stdout}
+
+	oldState, err := term.MakeRaw(int(os.Stdin.Fd()))
+	if err != nil {
+		return "", err
+	}
+	defer term.Restore(0, oldState)
```

Comment:
The godoc recommends not to use 0.
https://pkg.go.dev/golang.org/x/term#pkg-overview

```suggestion
	defer term.Restore(int(os.Stdin.Fd()), oldState)
```